### ISSUE TITLE
post_build_step.bat fires & extension launches even when extension is excluded from config

### ISSUE THREAD

**Issue by patchuby** [YoYoGames/GMEXT-EpicOnlineServices#8]
Running from IDE, in debug mode, on Windows, on 2024.2

Here is the build log
```C:\GMS\extensions\EpicOnlineServices\post_build_step.bat 
[EPICONLINESERVICES] INIT: Script initialization succeeded (v1.4.1 :: 1).
"Copying Windows (64 bit) dependencies"


    Directory: C:\Users\tomal\AppData\Local\GameMakerStudio2\GMS2TEMP


Mode                 LastWriteTime         Length Name                                                                 
----                 -------------         ------ ----                                                                 
d-----        14/05/2024     12:17                Chronique_des_Silencieux_409CCDB3_VM                                 




C:\GMS\extensions\EpicOnlineServices\post_build_step.bat DONE (0)
Igor complete.
[Run] Run game
[Debugger]target=127.0.0.1,yydebug="C:\Users\tomal\AppData\Local\GameMakerStudio2\GMS2TEMP\Chronique_des_Silencieux_409CCDB3_VM\Chronique des Silencieux.yydebug"
C:\ProgramData/GameMakerStudio2/Cache/runtimes\runtime-2024.2.0.163/windows/x64/Runner.exe  -game "C:\Users\tomal\AppData\Local\GameMakerStudio2\GMS2TEMP\Chronique_des_Silencieux_409CCDB3_VM\Chronique des Silencieux.win"
Setting scheduler resolution to 1
[EPIC ONLINE SERVICES]: Starting initialization...
[EPIC ONLINE SERVICES]: Current working dir: C:\Users\tomal\AppData\Local\GameMakerStudio2\GMS2TEMP\Chronique_des_Silencieux_409CCDB3_VM
[EPIC ONLINE SERVICES]: EOS_Platform_Create :: Platform creation error (check extension configuration)
```
Extension config : 
<img width="394" alt="image" src="https://github.com/YoYoGames/GMEXT-EpicOnlineServices/assets/9586594/c90be878-af9c-4fca-b636-8aee844cc119">

**Comment by patchuby** [comment]
I forgot : Deployment ID and Encryption Key are left empty in the Extension config as I didn't use them for the BuildPatchTool and the wiki didn't specify.

**Comment by DiasFranciscoA** [comment]
We were not able to reproduce this issue. Will will most likely need more information on it.

**Comment by patchuby** [comment]
This has been solved and was maybe due to IDE issues around setting up configs (fixed manually since I reported the issue).
Given there were fixes on that front in 2024.4 it seems, I think it's safe to close the issue, until and if I encounter the issue again.
